src/prune_builds

```diff
@@ -99,20 +131,22 @@ if skip_pruning:
 
 # now delete other build dirs not in the manifest
 
-for build_dir, _ in builds_to_delete:
+for build_dir, _, _ in builds_to_delete:
     shutil.rmtree(os.path.join(builds_dir, build_dir))
 
-
 # and finally prune OSTree repos
-def ostree_prune(repo_name, sudo=False):
-    repo = os.path.join(args.workdir, repo_name)
-    argv = []
-    if sudo:
-        argv.extend(['sudo', '--'])
-    print(f"Pruning {repo_name}")
-    argv.extend(["ostree", "prune", "--repo", repo, "--refs-only",
-                f"--depth={args.keep_last_n-1}"])
-    subprocess.run(argv, check=True)
-
-
-ostree_prune('repo')
+print(f"Pruning repo")
+repo = os.path.join(args.workdir, 'repo')
+
+# For now, we just use the --keep-younger-than CLI here. Doing this more
+# accurately would require enhancing the `prune` CLI (or just use the API
+# directly?). Or we could also manually figure out the depth of the oldest
+# build and then use that as the arg to `--depth`.
+oldest_ostree_t = new_builds[-1][2]
```

Comment:
Optional: Switch to e.g. `collections.namedtuple` as I had to go look up what `[2]` was.

Reply:
Heh, I literally have that in my diff while working on adding a `--keep-younger-than`. Will backport to this PR.